pyerrors/fits.py

```diff
@@ -130,6 +161,10 @@ def func(a, x):
     '''
     if priors is not None:
         return _prior_fit(x, y, func, priors, silent=silent, **kwargs)
+
+    elif (type(x) == dict and type(y) == dict and type(func) == dict):
+        return _combined_fit(x, y, func, silent=silent, **kwargs)
+
```

Comment:
Fine for now, in the future we have to cover all possible cases.

Reply:
We could catch the case where undefined behavior will happen, i.e., when the user does not supply two lists and a function or three dicts. This could help to identify the mistake. Otherwise, some error will be thrown in `_standard_fit` that is not necessarily well understandable.